src/App.js

```diff
@@ -64,7 +65,7 @@ class App extends Component {
     return (
       <div className="App">
         <SearchBar value={this.state.searchText} onChange={this.handleSearchBarChange.bind(this)}/>
-        <ContactList contacts={this.getFilteredContacts} />
+        <ContactList contacts={this.getFilteredContacts()} />
```

Comment:
You want to actually call this function to produce the list of contacts, so you need a `()`, unlike in an event handle where you leave off the `()` because you want to wait to call the function until it is triggered